load-tests/simple_producer_consumer/scripts/load_test/main.bal

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/log;
+import ballerina/http;
+import ballerina/lang.runtime;
+import ballerina/io;
+import ballerina/time;
+
+public function main(string label, int users, string output_csv_path) returns error? {
```

Comment:
What does this `users` argument do ?

Reply:
The result csv contains the user count. For now, "1" is passed for this

Reply:
So if we pass 2, does it create 2 producers ?

Reply:
no
this is just to create the csv file. it does not affect the kafka clients

Reply:
In that case, lets remove it from the arguments.